contracts/protocol/tokenization/NTokenMoonBirds.sol

```diff
@@ -32,6 +32,15 @@ contract NTokenMoonBirds is NToken, IMoonBirdBase {
         // Intentionally left blank
     }
 
+    function getNTokenIdentifier()
+        external
+        pure
+        override
+        returns (string memory)
+    {
+        return "NTokenMoonBirds";
```

Comment:
is identifier already set when we `initReserves` ?

https://github.com/para-space/paraspace-deploy/blob/07b8cb8a3b12b1dd8af0a51fe15f58a5d171ef51/helpers/init-helpers.ts#L319-L323